src/brpc/policy/baidu_rpc_protocol.cpp

```diff
@@ -69,7 +69,7 @@ inline void PackRpcHeader(char* rpc_header, int meta_size, int payload_size) {
 
 static void SerializeRpcHeaderAndMeta(
     butil::IOBuf* out, const RpcMeta& meta, int payload_size) {
-    const int meta_size = meta.ByteSize();
+    const int meta_size = meta.ByteSizeLong();
```

Comment:
Thanks for your contribution!
ByteSizeLong() is not available in old version protobuf, for compatibility with old version protobuf, you should test protobuf version here.